canonical-websites/jp.ubuntu.com
# Update the navigation styling to match the new designs

## Done
Update the navigation styling to match the new designs and added a new "English" link to the navigation on smaller screens.

## QA
- Check that the links match the [UX task](https://github.com/ubuntudesign/web-squad/issues/894#issuecomment-440299023)
- Check that the design matches [the design](https://github.com/canonical-websites/jp.ubuntu.com/issues/58)
- Check all screen sizes

## Issue / Card
Fixes https://github.com/canonical-websites/jp.ubuntu.com/issues/59
